src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1144,8 +1144,8 @@
     <Node Name="BoundTupleCreationExpression" Base="BoundExpression">
     <!-- Non-null type is required for this node kind -->
     <Field Name="Type" Type="TypeSymbol" Override="true" Null="disallow"/>
-    <!-- The constructor of the underlying tuple type -->
-    <Field Name="ConstructorOpt" Type="MethodSymbol" Null="allow"/>
+    <!-- The constructor(s) of the underlying tuple type. One for short tuples. Multiple for long tuples. (largest first) -->
+    <Field Name="ConstructorChainOpt" Type="ImmutableArray&lt;MethodSymbol&gt;" Null="allow"/>
```

Comment:
We had issues when types were there, but constructors missing. Ideally one should get a missing member error during binding, so we need to check if ctor is present.
Now it becomes a burden to keep all the constructors around. So, yes, perhaps we should check for the presence of the ctors, but not keep them, and mrk node as hasErrors if any one is missing so we would not try to lower. . When needed ast lowering time, we can find the ctors again.